cluster-up/cluster/ocp-4.3/provider.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-image="openshift-cnv/kubevirtci-ocp-4.3@sha256:446265401bff3e90548f34461f0814493f52489c82e4527844250f7a02dcbb7c"
+image="ocp-4.3@sha256:63bc365f97acab249b34a048825edd650973764355b3f232db7d74f7d08e6ea4"
```

Comment:
Let's just regenerate this with the prow jobs that have just merged, drop this for now.